pandas/tests/io/parser/test_read_csv.py

```diff
@@ -0,0 +1,154 @@
+import pytest
+import csv
+import os
+import sys
+
+rootpath = os.path.dirname(os.path.abspath(__file__))
+
+sys.path.append(os.path.join(rootpath, "pandas/io"))
+from pandas.io import parsers
+
+TMP_PATH = "tmp"
+
+"""
+To run test, run 'python path/to/test_parsers.py'
```

Comment:
we run all our tests with pytest, so this is unnecessary (as is the sys.path.append)